forge-core/src/main/java/forge/card/CardEdition.java

```diff
@@ -555,6 +555,20 @@ public List<PrintSheet> getPrintSheetsBySection() {
             }
 
             sheets.add(sheet);
+
+            String code2 = this.getCode2();
```

Comment:
Hmn, I don't think duplicating static data like that is a smart choice memory wise
shouldn't we have a global function that translates these back to the main code instead?